mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -127,6 +150,11 @@ final class Word2Vec @Since("1.4.0") (
     @Since("1.4.0") override val uid: String)
   extends Estimator[Word2VecModel] with Word2VecBase with DefaultParamsWritable {
 
+  private val batchSize = 10000
+  private val power = 0.75
+  private val maxUnigramTableSize = 100*1000*1000
+  private val unigramTableSizeFactor = 20
```

Comment:
Please add some comments describing these values. Should any of them be tune-able?